cmd/util/cmd/execution-state-extract/execution_state_extract.go

```diff
@@ -98,6 +98,9 @@ func extractExecutionState(
 				log,
 				nWorker,
 				[]migrators.AccountBasedMigration{
+					// Contracts must be migrated first
+					migrators.NewStagedContractsMigration(migrators.GetStagedContracts),
```

Comment:
I added this for the moment. But we would need to run the staged-contracts migration for all accounts first, before running the value migrations for any account. Not sure how to do it in a single binary. Can add it in a separate PR.

Reply:
cc @janezpodhostnik